src/org/sosy_lab/java_smt/api/FloatingPointFormulaManager.java

```diff
@@ -274,8 +275,8 @@ FloatingPointFormula castFrom(
 
   /**
    * Create a formula that interprets the given bitvector as a floating-point value in the IEEE
-   * format, according to the given type. The sum of the sizes of exponent and mantissa of the
-   * target type plus 1 (for the sign bit) needs to be equal to the size of the bitvector.
+   * format, according to the given type. The sum of the sizes of exponent and mantissa (including
+   * the hidden bit) of the target type needs to be equal to the size of the bitvector.
```

Comment:
This is wrong, right? It needs to refer to the sign bit, but exclude the hidden bit.